fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergUtil.java

```diff
@@ -353,11 +358,26 @@ public static Type convertColumnType(org.apache.iceberg.types.Type icebergType)
                 } else {
                     return Type.UNKNOWN_TYPE;
                 }
+            case MAP:
+                Type mapType = convertToMapType(icebergType);
+                if (mapType.isMapType()) {
+                    return mapType;
+                } else {
+                    return Type.UNKNOWN_TYPE;
+                }
+            case STRUCT:
+                List<Types.NestedField> fields = icebergType.asStructType().fields();
+                Preconditions.checkArgument(fields.size() > 0);
+                ArrayList<StructField> structFields = new ArrayList<>(fields.size());
+                for (Types.NestedField field : fields) {
+                    String fieldName = field.name();
+                    Type fieldType = convertColumnType(field.type());
```

Comment:
Add a check, if fieldType is UNKNOWN_TYPE, we should return UNKOWN_TYPE directly.

Struct doesn't support unknown type's subfield.

Reply:
GET